app/authentication/token.py

```diff
@@ -91,3 +116,27 @@ async def get_user(token: str = Depends(oauth2_scheme)) -> Tuple[str, str]:
         raise HTTPException(status_code=401, detail="Unauthorized")
     else:
         return response.json()["id"], response.json()["role"]
+
+
+async def user_is_owner_or_admin(dataset: str) -> bool:
+    user_id, role = await get_request_user()
+
+    if role == "ADMIN":
```

Comment:
Not sure if we still want to go for this, but technically we check if ADMIN is also "gfw" application. Did we still want to check an application, or should RW ADMINs just have universal access?